spacy/tests/pipeline/test_pipe_methods.py

```diff
@@ -605,10 +605,35 @@ def component(doc):
             assert results[component] == ""
 
 
-def test_load_disable_enable() -> None:
-    """
-    Tests spacy.load() with dis-/enabling components.
-    """
+@pytest.mark.issue(11443)
+def test_issue11443():
```

Comment:
Let's make the name of this test descriptive (we've moved away from `test_issueXXX` as we've got the pytest markers now)